projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+      }
+   ]
+}
+```
+
+You can now store the identifier of the organizer "Natuurpunt Holsbeek" (`e83edb7f-6f23-47cd-a730-66fd7687f116`) in your application to reuse it as organizer ID in the `POST` request of the event(s) that you want to create.
+
+### Searching existing organizers within the entry form
+
+In this solution you directly integrate with the UiTdatabank Search API in your own entry form.
+
+* ✅ Suitable when the organizers for the events created through your application are unpredictable in advance
+* ❌ Not recommended when the events created through your application all have the same organizer
+* ❌ Not recommended when the events created through your application can have a different organizer, but (1) the amount of organizers is limited & (2) the list of organizers is static and won't change a lot over time
+
+#### Example
+
+In the example below, a user in your entry form types the name "Natuurpunt Holsbeek" in the input field for the organizer. Under the hood an API call to the UiTdatabank Search API is made to check if that organizer already exists:
+
+**request**
+
+```
+GET /organizers/?q=name.\*:"Natuurpunt Holsbeek"
+```
+
+**response**
+
+The response now contains the identifier of "Natuurpunt Holsbeek":
+
+```js
+{
+   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
+   "@type":"PagedCollection",
+   "itemsPerPage":30,
+   "totalItems":1,
+   "member":[
+      {
+         "@id":"https://io.uitdatabank.be/organizers/e83edb7f-6f23-47cd-a730-66fd7687f116",
+         "@type":"Organizer"
+      }
+   ]
+}
+```
+
+If the organizer already exists (like in the example above), it is suggested in the user interface in a human-readable way so that the user can select the organizer and thus choose to reuse the organizer for the creation of his or her event.
+
+Only when no results are found, or the results found do not match the organizer that the user wants to use, a new organizer can be created by the user.
+
+### Provide a mapping when exporting your data to UiTdatabank
+
+In this solution you add a check when exporting the event data to UiTdatabank. The export can take place at a different (later) time than when the event was created by the user in the own application.
+
+* ✅ Suitable when the organizers for the events created through your application are unpredictable in advance
+* ❌ Not recommended when the events created through your application all have the same organizer
+* ❌ Not recommended when the events created through your application can have a different organizer, but (1) the amount of organizers is limited & (2) the list of organizers is static and won't change a lot over time
+
+In this solution you save the relevant data for organizers inside your own application / database. Before exporting the event data to UiTdatabank a check is made to see if the organizers linked to the events already exist in UiTdatabank or not.
+
+This check can be done on the following fields:
+
+* `name`
```

Comment:
Maybe we should limit this to just `website`? That field is required anyway if they want to create a new organizer (so they need have it either way) and more reliable than name?

Reply:
I'm not sure a website is more reliable, just like the `name` a link can have many variants, e.g. the link of an organizer can be something like `facebook.com/natuurpunt-holsbeek` but also `natuurpunt.be/afdelingen/natuurpunt-holsbeek`.

Reply:
Yes but what I meant with "reliable" is that the user will need to enter a website URL anyway (because it's a required field for creating a new organizer as well), and when they later try to map the data to UDB they have to check if an organizer with that website exists or not because it's technically impossible to create a new one with the same website. So they have to do that check already anyway. If they'd do a check on name instead it's possible that they do not find the organizer and then try to create it, only to get an error that an organizer with the same website already exists. So it's more reliable in the sense that if you check the website and you find 0 results, you will be able to create a new one. And it's still a good effort of trying to re-use organizers I think? Since it's also possible that they get 0 matches for the name while the same organizer already exists (for example shortly after a rename like Vooruit to Viernulvier, or with a slightly different/incorrect notation like "Vier-nul-vier" instead of "viernulvier")

Reply:
I think not all integrators ask users in their entry form to add an url for an organizer. It is for example possible that the user in the user interface can only enter a name for the organizer.
It is also possible that, when no existing organizer is found in UiTdatabank, the organizer property is omitted in the `POST` request for the event. 

We're both making assumptions about how the integrator integrates with the Entry API ofc..

Reply:
Slightly related: from my experience entry api integrations of partners are rather basic (sometimes too basic) set up:
- Peppered creates organizer X and place Y manually on www.uitdatabank.be before they set up an integration for organizer X or place Y
- Gezinsbond / Cegeka only makes it possible to create an event for example. All updates on an event should be done in the ui of UiTdatabank (sad but true)
- Some integrators even don't keep the identifiers of the events / places / organizers they create in their own DB (`PWK-769` as example or mor recently `III-4905`)
- ..

Therefore I think it is unlikely that many integrators really ask the url for an organizer in their own entry form, especially since that `url` is probably only necessary for UiTdatabank and not for other purposes.

Reply:
But then how would they create a new organizer (instead of reusing an existing one) in the first place? Since the organizer website is a required field to create a new organizer?

Reply:
Clarified on Slack that the guide is about reusing existing organizers in general and not only for preventing duplicates. In that case this makes sense indeed 👍